docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+	            console.log("Game Over");
+	        } else {
+	            console.log('Tweeted: ' + reply.text);
+	        }
+	    }
+	}
+
+	 const job1 = schedule.scheduleJob(rule, pressSelect);
+
+	 job1.on("Every Day Tweet", pressSelect);
+```
+</br>
+
+Here is the function used to tweet on a schedule, which I’ve named `pressSelect`. `pressSelect` has the `replyText` variable, slightly changed to be an original tweet rather than a comment on a different tweet, but pulls from the same array. The `gameOver` function is also present, though renamed just to be safe. Since `gameOver` and `gameOver2` are local variables within their respective functions, there shouldn’t be any issues. However, I'm just being safe.
+
+The final part of this code is creating a variable called `job1`. `job1` is the `scheduleJob` method from the node-schedule object at the top of the code. I pass in `rule` created earlier in the code and `pressSelect` as parameters. We then use an event listener with `pressSelect` passed in again. That's it for the code.
+
+## Running the Code
+
+To test your code and ensure it works, type `npm run develop` into the terminal. If you get a Twit error about consumer keys, ensure there are no spaces between the variable, equals sign, and key itself in your .env file. If the error persists, you may have copied your keys wrong. You can always generate them again and copy them directly into the .env file. If you’d like to test `pressSelect` on its own and make sure it works, you can just comment out the last two lines of the code and call `pressSelect ` directly. Under the commented out code, just type
+
+```
+
+	pressSelect();
+
+```
+and the function will run. This way, you don’t have to wait for whatever day and hour you scheduled node-schedule for.
+
+Once the bot is running, if you want to test the way it responds to other users, log in to another Twitter account and tweet at your bot. You should be able to see some action in the terminal that tells you it’s working, followed by the response on Twitter.
+
+## Hosting Your Code
+
+You could certainly let this code run for a long while from your local machine, but it’d be better if you had it hosted somewhere else. One great option is Linode.
+
+After signing up for a Linode account, the home page will look like this.
+
+![Linode home screen](https://lh3.googleusercontent.com/JFNpdFMCe9A37beAwtxazN-zqcSr88Ff457bnQhbQpkQJILfqAv7g0bR_CQ6SxMu8EfKgIcaqTGuZvPTTI2hOb6dYyi3CyLMubEKOwFEZMkCaByjpk83L2o0c4W8GTwE4VPSodE-)
+</br>
+
+Create a Linode, and pick Ubuntu 20.14 from the first dropdown menu. Select the region closest to you and then choose Nanode, the smallest option. It’s only $5 a month and has more than enough space and RAM for your Twitter bot and any other projects you’d want on it. When it asks for an Ubuntu label, name it after your project. Create a root password and click create. Make sure to remember the root password as you’ll have to use it every time to get into the Linode.
```

Comment:
I figured Ubuntu 20.14 was a typo for Ubuntu 20.04, but we can switch this up if you intended for a different version